release-notes/EPM/I-DOCSIS/EPM_7.0.5_I-DOCSIS.md

```diff
@@ -4,6 +4,20 @@ uid: EPM_7.0.5_I-DOCSIS
 
 # EPM 7.0.5 I-DOCSIS
 
+## New features
+
+#### New dashboards for fiber node utilization reporting [ID_40111]
```

Comment:
@EstebanSkyline, could you add these new dashboards to the [list in the user guide](https://docs.dataminer.services/user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_components.html#dashboards)?